action.yml

```diff
@@ -19,9 +19,8 @@ inputs:
     description: Environment to deploy (required)
     required: true
   slack-webhook:
-    description: Slack Webhook URL, will not notify if empty
-    required: true
```

Comment:
You don't have to drop this (though it's fine to). The interaction of `required` and `default` is odd.

`required` means `getInput()` / `${{ inputs. }}` will error if it's not present (vs giving `null`). And `default` ensures it's present even if the user didn't provide it. So `required` with `default` is fine, and seems to be how most folks do it.